readycloud/readycloud.py

```diff
@@ -21,10 +21,19 @@ class ReadyCloud(object):
     :type token: str
     :param host: host with which you want to work (readycloud.com by default)
     :type host: str
+    :param api: api version (v2 by default)
+    :type host: str
     """
```

Comment:
We should also have  a description of the `org_id` param. It's not just the org's p, it should be hexahexacontanecimal encoded.